tests/lib/devicetree/api/testcase.yaml

```diff
@@ -3,5 +3,4 @@ tests:
     tags: devicetree
     # We only need this to run on one platform so use native_posix as it
     # will mostly likely be the fastest.
-    integration_platforms:
-      - native_posix
+    platform_allow: native_posix qemu_x86 qemu_x86_64 qemu_cortex_m3
```

Comment:
Nothing wrong with this line, but the commit message contains this, which should be removed:

```

fix
```